config/application.yml.default

```diff
@@ -196,6 +196,7 @@ rules_of_use_updated_at: '2021-05-21T00:00:00Z'
 s3_public_reports_enabled: 'false'
 s3_reports_enabled: 'false'
 saml_secret_rotation_enabled: 'false'
+seed_agreements_data: 'false'
```

Comment:
Do we need a top-level key as the default value for dev, test, etc?

Reply:
Do you mean  because `nil` defaults to false? Or something else?

Reply:
I got confused about where this was in the structure of the file.

I think at the top-level it should be `true`
and then we should override it in the `production:` block to be `false`

and then like you said, override it via the value in S3 again in prod

Reply:
Got it, yeah that makes sense. I'll make that update and hopefully that resolves the failing specs.